spk/gerrit/src/dsm-control.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+
+# Package
+PACKAGE="gerrit"
+DNAME="Gerrit"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+PATH="${INSTALL_DIR}/bin:${GIT_DIR}/bin:${PATH}"
+USER="gerrit"
+PID_FILE="${INSTALL_DIR}/logs/gerrit.pid"
+LOG_FILE="${INSTALL_DIR}/logs/error_log"
+
+start_daemon ()
+{
+    su ${USER} -c "PATH=${PATH} ${INSTALL_DIR}/bin/gerrit.sh start"
```

Comment:
Was just asking because it isn't listed in `cross/gerrit` PLIST and the digests file also only refers to the .war file ...